recipes/libpq/all/patches/13.3/001-Disable-thread-safety-check-under-thread-sanitizer.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/test/thread/thread_test.c b/src/test/thread/thread_test.c
+index e1bec01..e4ffd78 100644
+--- a/src/test/thread/thread_test.c
```

Comment:
This is a patch for a test.
Does libpq build/run tests?

Reply:
This test is run during the configure phase. This is not exactly a test, this is more of a feature check.